.gitignore

```diff
@@ -1,10 +1,10 @@
-Meta
-doc
-issues
-.Rproj.user
-.Rhistory
-.RData
-inst/doc
-temp.R
-tags
-README.html
+Meta

```

Comment:
Looks like we don't use the same line endings.  I'm surprised git hasn't dealt with this automatically.